PHP/Endpoint Examples/Multipart Payload/pdf-with-redacted-text-preview.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require 'vendor/autoload.php'; // Require the autoload file to load Guzzle HTTP client.
+
+use GuzzleHttp\Client; // Import the Guzzle HTTP client namespace.
+use GuzzleHttp\Psr7\Request; // Import the PSR-7 Request class.
+use GuzzleHttp\Psr7\Utils; // Import the PSR-7 Utils class for working with streams.
+
+$client = new Client(); // Create a new instance of the Guzzle HTTP client.
+
+$headers = [
+  'Api-Key' => 'xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' // Set the API key in the headers for authentication.
+];
+
+$options = [
+  'multipart' => [
+    [
+      'name' => 'file', // Specify the field name for the file.
+      'contents' => Utils::tryFopen('/path/to/file', 'r'), // Open the file specified by the '/path/to/file' for reading.
+      'filename' => '/path/to/file', // Set the filename for the file to be processed, in this case, '/path/to/file'.
+      'headers' => [
+        'Content-Type' => '<Content-type header>' // Set the Content-Type header for the file.
+      ]
+    ],
+    [
+      'name' => 'redactions', // Specify the field name for the text options.
+      'contents' => '[{"type":"preset","value":"uuid"},{"type":"regex","value":"(\\+\\d{1,2}\\s)?\\(?\\d{3}\\)?[\\s.-]\\d{3}[\\s.-]\\d{4}"},{"type":"literal","value":"word"}]' // Set the value for the redactions option. This is a JSON-formatted string consisting of an array with sets of text options.
```

Comment:
Getting a JSON formatting error here
```
{"error":"The JSON data provided is not properly formatted. Please check your syntax and try again."}
```

The `uuid` error likely applies here too.
```suggestion
      'contents' => '[{"type":"preset","value":"email"},{"type":"regex","value":"(\\+\\d{1,2}\\s)?\\(?\\d{3}\\)?[\\s.-]\\d{3}[\\s.-]\\d{4}"},{"type":"literal","value":"word"}]' // Set the value for the redactions option. This is a JSON-formatted string consisting of an array with sets of text options.
```